xcm/xcm-executor/src/lib.rs

```diff
@@ -83,15 +83,44 @@ impl<Config: config::Config> ExecuteXcm<Config::Call> for XcmExecutor<Config> {
 		);
 		let xcm_weight = match Config::Weigher::weight(&mut message) {
 			Ok(x) => x,
-			Err(()) => return Outcome::Error(XcmError::WeightNotComputable),
+			Err(()) => {
+				log::debug!(
```

Comment:
y u no trace! here?

Reply:
Cause I was thinking that it would be nice to get the most important subset of the logging via `-lxcm=debug`